libgstc/python/pygstc/gstc.py

```diff
@@ -1101,6 +1101,34 @@ def signal_timeout(
             [pipe_name, element, signal, timeout], [str, str, str, int])
         self._send_cmd_line(['signal_timeout'] + parameters)
 
+    def action_emit(self, pipe_name, element, action):
+        """
+        Emits an action with no-parameters
+
+        Parameters
+        ----------
+        pipe_name: string
+            The name of the pipeline
+        element: string
+            The name of the element
+        action: string
+            The name of the action
+
+        Raises
+        ------
+        GstdError
+            Error is triggered when Gstd IPC fails
+        GstcError
+            Error is triggered when the Gstd python client fails internally
+        """
+
+        self._logger.info(
+            'Triggering action %s of element %s from pipeline %s'
+            % (action, element, pipe_name))
```

Comment:
I changed all logs in [this commit](https://github.com/RidgeRun/gstd-1.x/pull/263/commits/62b303bd1322f0a500e2394c6047fdf26ea34fe8) to use the preferred way of formatting strings in Python 3.5 (.format)
Please change this log to use that style.

Reply:
I didn't notice that. I will do it right-away

Reply:
I see it is pending. Still, I will change it looking forward of your merge

Reply:
I saw this:
https://realpython.com/python-f-strings/